### ISSUE TITLE
HeaderTooltips `onlyTrimmed` works wrong with header texts longer than 50px

### ISSUE THREAD

**Issue by warpech** [handsontable/handsontable#6708]
### Description

According to the HeaderTooltips docs page (https://handsontable.com/docs/7.4.0/HeaderTooltips.html):

> the `onlyTrimmed` property defines if tooltips should be added only to headers, which content is trimmed by the header itself (the content being wider then the header).

However, actually if the header text is wider than 50px, the tooltip is always shown, even if the text fits well within the header cell.

I have investigated a bit, and the problem comes from the fact that HeaderTooltips plugin useses the `afterGetColHeader` hook, which is fired BEFORE the `width` is applied to the `<col>` element. That comes from the fact that in Walkontable, `columnHeaders.render()` is called before `colGroup.render()`.

The only reason why `onlyTrimmed`  works for texts shorter than 50px is because `col` has a default width of 50px set in the CSS.

### Steps to reproduce

1. Go to https://jsfiddle.net/36sLkcuw/
2. Put your mouse cursor above the header cell "Demo"
3. Notice that you can see the tooltip (this is expected)
4. Go to https://jsfiddle.net/cha52mrL/
5. Put your mouse cursor above the header cell "Demogorgon"
6. Notice that you can see the tooltip (this is unexpected)

### Demo

 https://jsfiddle.net/cha52mrL/

### Your environment
* Handsontable version: 7.4.0, 7.3.0
* Browser Name and version: any
* Operating System: any

![image](https://user-images.githubusercontent.com/566463/74418075-17ac4880-4e48-11ea-8fe8-7d839a2e5d70.png)

**Comment by warpech** [comment]
After some more investigation, it looks like the problem is caused by interaction of the HeaderTooltips plugin with AutoColumnSize plugin.

I pushed a speculative fix in 0f2fd8ffccb27c908342fec8ba5a887d988dcffb. I discussed this fix with @wojciechczerniak, and he suggested that it is not desired, because the `afterRender` hook should not be used to add more manipulations to DOM. 

To do it property, a new plugin hook `afterColRenderer` could be used. Or we can make changes so that `colGroup.render()` runs BEFORE `columnHeaders.render()`.
